com.kms.cura.dal/src/com/kms/cura/dal/database/DatabaseHelper.java

```diff
@@ -140,4 +140,26 @@ public void insertReferenceRowsToReferenceTable(String database, List<ReferenceT
 			}
 		}
 	}
+	
+	public Entity queryEmailPasswordJoin(String tableName1, String tableName2, String email, String password, String onTable1, String onTable2) throws SQLException, ClassNotFoundException{
+		PreparedStatement stmt = null;
+		ResultSet rs = null;
+		try {
+			stmt = con.prepareStatement(
+					"SELECT * FROM " + tableName1 + " LEFT JOIN " + tableName2 + " ON " + tableName1+"."+onTable1 + " = " + tableName2 + "."+ onTable2 +
```

Comment:
There's a lot of string concatenation here. Should change to using StringBuilder